airbyte-integrations/connectors/source-auth0/source_auth0/manifest.yaml

```diff
@@ -0,0 +1,125 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: []
+
+  requester:
+    type: HttpRequester
+    url_base: "{{ config['base_url'] }}"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['credentials']['access_token'] }}"
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "#/definitions/requester"
+
+  base_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/retriever"
+
+  # incremental_base:
+  #   type: DatetimeBasedCursor
+  #   cursor_field: "updated_at"
+  #   datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z"
+  #   cursor_granularity: "PT0.000001S"
+  #   lookback_window: "P31D"
+  #   start_datetime:
+  #     datetime: "{{ config['start_date'] }}"
+  #     datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z"
+  #   end_datetime:
+  #     datetime: "{{ today_utc() }}"
+  #     datetime_format: "%Y-%m-%d"
+  #   step: "P1M"
+
+  clients_stream:
+    $ref: "#/definitions/base_stream"
+    name: "clients"
+    primary_key: "client_id"
+    $parameters:
+      path: "clients"
+
+  users_stream:
+    $ref: "#/definitions/base_stream"
+    name: "users"
+    # incremental_sync:
+    #   $ref: "#/definitions/incremental_base"
+    primary_key: "user_id"
+    $parameters:
+      path: "users"
+
+  organizations_stream:
+    $ref: "#/definitions/base_stream"
+    name: "organizations"
+    primary_key: "id"
+    $parameters:
+      path: "organizations"
+
+  organization_members_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/organizations_stream"
+        parent_key: "id"
+        partition_field: "parent_id"
+
+  organization_members_stream:
+    $ref: "#/definitions/base_stream"
+    name: "organization_members"
+    primary_key: "user_id"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        path: "organizations/{{ stream_partition.parent_id }}/members"
```

Comment:
@sajarin Could you please help with the parent stream slice? There is a key error for this stream, but I've done everything correctly.

Reply:
Ping via slack for the credentails